packages/http/src/validator/validators/security/handlers/bearer.ts

```diff
@@ -0,0 +1,21 @@
+import { fromNullable, getOrElse, map } from 'fp-ts/lib/Option';
+import { pipe } from 'fp-ts/lib/pipeable';
+import { get, partial } from 'lodash';
+import { when } from './utils';
```

Comment:
Do we still practice separating internal and external imports?

Reply:
yes, let me fix this.